definitions/procedures/katello_service/restart.rb

```diff
@@ -0,0 +1,17 @@
+module Procedures::KatelloService
+  class Restart < ForemanMaintain::Procedure
+    metadata do
+      description 'katello-service restart'
+      param :only, 'A comma-separated list of services to include', :array => true
+      param :exclude, 'A comma-separated list of services to skip', :array => true
+    end
+
+    def run
+      with_spinner('restarting katello service(s)') do |spinner|
```

Comment:
Are `meth_call_by_action` and `run_with_hammer_ping_retry` necessary. What if we did something like:

```
spinner.update("Restarting services")
feature(:katello_service).restart(:only => @only, :exclude => @exclude)
feature(:katello_service).hammer_ping_retry(spinner)
```